src/commands/install/createCypressConfig.js

```diff
@@ -0,0 +1,39 @@
+const inquirer = require('inquirer')
+const { write } = require('../../utils/fs.js')
+const { CYPRESS_CONFIG_PATH } = require('../../utils/paths.js')
+
+const createCypressConfig = async force => {
+    const prompt = inquirer.createPromptModule()
+
+    const cypressAnswers = await prompt([
+        {
+            type: 'input',
+            name: 'baseUrl',
+            message: 'URL that Cypress should run tests against:',
+            default: 'http://localhost:3000',
+        },
+        {
+            type: 'confirm',
+            name: 'video',
+            message: 'Record video?',
+            default: false,
+        },
+        {
+            type: 'input',
+            name: 'projectId',
+            message: 'The unique Cypress project ID:',
+        },
+    ])
+
+    write(
+        CYPRESS_CONFIG_PATH,
+        {
+            baseUrl: cypressAnswers.baseUrl,
+            video: cypressAnswers.video,
+            ...(cypressAnswers.projectId ? [cypressAnswers.projectId] : []),
```

Comment:
@varl This should be `cypressAnswers.projectId ? { projectId: cypressAnswers.projectId } : {}`, shouldn't it?
I'll test on Tuesday what'll happen when I provide a value for projectId, guess none of use actually tried to use this one so far.

Which leads me to another thought (cc @amcgee): Would it make sense to have integration tests for our cli libraries (i. e. creating a temp dir, `yarn init`, `d2 utils cypress install`, answer the prompts depending on what the test wants to test and finally check the result)?

I'm not sure tests are actually worth the time here. I guess once our CLI tools work, we kind of know that they work, they're not used in production apps anyways so if a bug exists, it's most like going to be nothing but a nuisance.

Reply:
@Mohammer5 I think that would be nice, at least for smoke tests, but I don't think it's highest priority as you mentioned - if things break, we will notice and can quickly fix them without much fuss since cli tools are continuously deployed

Reply:
But when we have some bandwidth I do think this is a very good idea, especially for tools that have implications for external developers making DHIS2 apps because bugs introduce friction there

Reply:
@Mohammer5 yeah I think this line `...(cypressAnswers.projectId ? [cypressAnswers.projectId] : [])` is incorrect though, needs to be an object otherwise it will set `0: cypressAnswers.projectId`

Reply:
Fixed